rdrf/registry/patients/parent_view.py

```diff
@@ -174,6 +174,13 @@ def get(self, request, registry_code, parent_id):
     def post(self, request, registry_code, parent_id):
         context = {}
         parent = ParentGuardian.objects.get(id=parent_id)
+        associated_parent = ParentGuardian.objects.filter(user=request.user).first()
+        if not associated_parent:
+            user = request.user
+            if not (user.is_superuser or self.request.user.is_staff):
```

Comment:
Too lax again.
Anyone with `is_staff` can edit parents? So a clinician for example with `is_staff` would be able to edit the parents in another registry?

Reply:
Updated.